docs/learning/tutorials/develop-web-viewer.md

```diff
@@ -5,6 +5,7 @@
 - [Install necessary prerequisites]($docs/getting-started/development-prerequisites).
 - From a terminal, `npx create-react-app your-app-name --template @bentley/itwin-viewer --scripts-version @bentley/react-scripts`.
   - This will generate a new application based on the iTwin Viewer React component in the `your-app-name` directory.
+- In the `your-app-name` directory, `npm install`.

```

Comment:
@NancyMcCallB you shouldn't need to call `npm install` here.  The create-react-app generator runs installation for you.

Reply:
Yep. I got an error that npm install fixed. I just did a new create-react-app, though, and it was fine. I'll remove it.

Reply:
Failed to compile.

./node_modules/@microsoft/applicationinsights-web/dist-esm/applicationinsights-web.js
Attempted import error: 'PerfEvent' is not exported from '@microsoft/applicationinsights-core-js'.

Reply:
What version of Node are you using?  I'll give it a test now.

Reply:
Also can you try doing `--use-npm`?